packages/ec-core/src/operator/selector/lexicase.rs

```diff
@@ -95,3 +100,181 @@ where
             .context("The pool of candidates was empty")
     }
 }
+
+#[cfg(test)]
+#[rustversion::attr(before(1.81), allow(clippy::unwrap_used))]
+#[rustversion::attr(
+    since(1.81),
+    expect(
+        clippy::unwrap_used,
+        reason = "Panicking is the best way to deal with errors in unit tests"
+    )
+)]
+mod tests {
+    use prop::collection;
+    use proptest::prelude::*;
+
+    use super::*;
+    use crate::individual::ec::EcIndividual;
+
+    #[test]
+    fn single_best_single_error() {
+        // 9 is the sole winner
+        let population = vec![5, 8, 9, 6, 3, 2, 0]
```

Comment:
don't use `vec![]` here - it's not needed, just use an array.

Reply:
Thanks for catching that. I started with an array, and for some reason decided I needed to change to `vec!`. It works fine as an array, though, so I don't know what I was thinking.